rocketpool-cli/wallet/utils.go

```diff
@@ -20,35 +26,65 @@ func promptPassword() string {
 		confirmation := cliutils.PromptPassword("Please confirm your password:", "^.*$", "")
 		if password == confirmation {
 			return password
-		} else {
-			fmt.Println("Password confirmation does not match.")
-			fmt.Println("")
 		}
+
+		fmt.Println("Password confirmation does not match.")
+		fmt.Println("")
 	}
 }
 
 // Prompt for a recovery mnemonic phrase
 func promptMnemonic() string {
 	for {
 		mnemonic := cliutils.PromptPassword("Please enter your recovery mnemonic phrase:", "^.*$", "")
+		// If the user copy-pasted their mnemonic, don't make them panic by failing on the UTF8 guardians
+		mnemonic = strings.ReplaceAll(mnemonic, UTF8_EMPTY_SPACE, "")
```

Comment:
latently added these two lines.

it seems like a user should be allowed to copy paste their mnemonic on creation, as long as they can manually enter it, and then if they need to recover, we shouldn't punish them.